tests/regression/lp-1898038/test-docker-support-app/bin/test-docker-support

```diff
@@ -0,0 +1,2 @@
+#!/usr/bin/env bash
+/sbin/apparmor_parser -r $SNAP/test-docker-support.profile
```

Comment:
You may need to quote that to make shellcheck happy

Reply:
Thanks, fixed in 5895a1e777